Recast/Views/HomeTableViewHeaderView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  HomeTableViewHeaderView.swift
+//  Recast
+//
+//  Created by Jaewon Sim on 9/17/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol HomeTableViewHeaderViewDelegate: class {
+    func homeTableViewHeaderDidPressSeeAll(sender: HomeTableViewHeaderView)
```

Comment:
homeTableHeader(_ homeTableHeader: HomeTableViewHeader, didPress: Action) 

Make and enum for these actions maybe in case we add actions later.